lib/re.js

```diff
@@ -23,7 +23,7 @@ module.exports = function (opts) {
   // All possible word characters (everything without punctuation, spaces & controls)
   // Defined via punctuation & spaces to save space
   // Should be something like \p{\L\N\S\M} (\w but without `_`)
-  re.src_pseudo_letter       = '(?:(?!' + text_separators + '|' + re.src_ZPCc + ')' + re.src_Any + ')';
+  re.src_pseudo_letter = '(?:(?!' + text_separators + '|' + re.src_ZPCc + ')' + re.src_Any + '|[a-z0-9]+_[a-z0-9]+)';
```

Comment:
IMO it's not good idea to cheat with letter definition. Dash should be added to upper level rules. `src_pseudo_letter` should be exactly as described - letter of word.

Reply:
> IMO it's not good idea to cheat with letter definition.

Fair.

> Dash should be added to upper level rules. src_pseudo_letter should be exactly as described - letter of word.

How were you seeing that? Should I add `re.letters_with_underscore = '[a-z0-9]+_[a-z0-9]+'` above or is there a regex in `uc.micro` that can be pulled in?

Reply:
I would update `re.src_domain_root` & `re.src_domain` directly. But prior to do that, worth to clarify why such fix needed at all.